scripts/node.sh

```diff
@@ -812,5 +855,9 @@ do
    esac || msg "node process finished with status $?"
    ${loop} || break
    msg "restarting in 10s..."
+   rm_bls_pass
    sleep 10
+   if ${multi_key}; then
```

Comment:
this won't work, the program will run in line 853 and 854.

Reply:
this is only for the failure case. non failure case is already considered.